### DISCUSSION TITLE
configure "invalid redirect uri" page

### DISCUSSION THREAD

**Discussion by Frank-LR** [keycloak/keycloak#27920] [Q&A]
Hi,

sometimes keycloak presents the user this error page (see screenshot)

- _We're sorry ..._
- _invalid redirect uri_

how / where can I configure, that keycloak redirects in this case to a special site of our app, where the user gets more information how to handle this?  (i.e. contact support)

![Keycloak invalid redirect url](https://github.com/keycloak/keycloak/assets/50948017/79817718-9f26-4d5a-b8a1-5f6bdd426dee)

**Comment by codespearhead** [comment]
By "sometimes" you mean that the valid redirect URIs are properly set in the client, but that "invalid redirect URI" shows up randomly?

If so, show us what's in the **Access settings** section of your client.
